robot_specific_config/abb_irb2600_12_185_moveit_config/config/joint_limits.yaml

```diff
@@ -0,0 +1,34 @@
+# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
+# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
+# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
+joint_limits:
+  joint_1:
+    has_velocity_limits: true
+    max_velocity: 5.027
```

Comment:
The `max_velocity` values in this file do not correspond to the limits set in the urdf. The values here can be lower than max limits specified in the urdf but not higher.

Reply:
Changed in [db0e85f](https://github.com/PickNikRobotics/abb_ros2/pull/53/commits/db0e85f3fc98db8a2c4ed112ef5893c488f2b4ed)